helpers/option_ext.rei

```diff
@@ -0,0 +1,9 @@
+include (module type of {
+  include Option;
+});
+module Let_syntax: {
+  let some: 'a => option('a);
+  let (let.none): (option('a), unit => option('a)) => option('a);
```

Comment:
Non blocking: Is this syntax actually useful? It's not very intuitive to me, unlike `let.some`. Maybe just use normal match in this case?

cc @EduardoRFS

Reply:
Done: https://github.com/marigold-dev/deku/issues/225